packages/eas-cli/src/commands/device/delete.ts

```diff
@@ -0,0 +1,184 @@
+          `Found ${appleDevices.length} devices for team ${appleTeamName ?? appleTeamIdentifier}`
+        );
+
+        Log.newLine();
+        if (udids) {
+          udids.forEach(udid => {
+            const foundDevice = appleDevices.find(device => device.identifier === udid);
+            if (foundDevice) {
+              chosenDevices.push(foundDevice);
+            } else {
+              Log.warn(`No device found with UDID ${udid}.`);
+            }
+          });
+        }
+
+        if (chosenDevices.length === 0) {
+          Log.addNewLineIfNone();
+          chosenDevices = await chooseDevicesToDeleteAsync(appleDevices);
+          Log.newLine();
+        }
+
+        Log.addNewLineIfNone();
+        chosenDevices.forEach(device => {
+          Log.log(
+            `${formatDevice(device, {
+              appleTeamName,
+              appleTeamIdentifier: appleTeamIdentifier!,
+            })}`
+          );
+          Log.newLine();
+        });
+
+        if (chosenDevices.length > 0) {
+          Log.warn(
+            `You are about to remove the Apple device${
+              chosenDevices.length > 1 ? 's' : ''
+            } listed above from your Expo account.`
+          );
+          Log.newLine();
+
+          const confirmed = await toggleConfirmAsync({
+            message: 'Are you sure you wish to proceed?',
+          });
+
+          if (confirmed) {
+            const removalSpinner = ora(`Removing Apple devices on Expo`).start();
+            try {
+              chosenDevices.forEach(async chosenDevice => {
+                await AppleDeviceMutation.deleteAppleDeviceAsync(chosenDevice.id);
+              });
+            } catch (err) {
+              removalSpinner.fail();
+              throw err;
+            }
+            removalSpinner.succeed('Removed Apple devices from Expo');
+
+            Log.newLine();
+            const deleteOnApple = await toggleConfirmAsync({
+              message: 'Do you want to remove these devices from your Apple account as well?',
+            });
+
+            if (deleteOnApple) {
+              // Delete on Apple?
+              const ctx = await authenticateAsync({ teamId: appleTeamIdentifier });
+              const context: RequestContext = getRequestContext(ctx);
+
+              Log.addNewLineIfNone();
+              const removeAppleSpinner = ora('Removing devices from Apple').start();
+              try {
+                const realDevices = await Device.getAllIOSProfileDevicesAsync(context);
+                realDevices
+                  .filter(d => chosenDevices.map(cd => cd.identifier).includes(d.attributes.udid))
+                  .forEach(device => {
+                    device.updateAsync({ status: DeviceStatus.DISABLED });
+                  });
+              } catch (err) {
+                removeAppleSpinner.fail();
+                throw err;
+              }
+              removeAppleSpinner.succeed('Removed devices from Apple');
```

Comment:
Maybe we should be explicit here. We actually don't **remove** the devices, we **disable** them.

Reply:
Change that in other logs as well.